sihl_core/lib/db.mli

```diff
@@ -45,6 +45,10 @@ module Migrate : sig
 
   type migration = string * migration_step list
 
+  module MariaDbRepository : Contract.Migration.REPOSITORY
```

Comment:
This and Postgresql one should not be here, but can be refactored out later.

Reply:
Agree, this is temporary.